controllers/review.js

```diff
@@ -0,0 +1,27 @@
+// require dependencies
+const express = require('express');
+const Review = require('../models/review');
+
+// initialize router
+const router = express.Router();
+
+// Create Route
+router.post('/', async (req,res) => {
+
+})
+
+// Get all reviews
+router.get('/', async (req, res) => {
+    try {
+      const reviews = await Review.find();
+      res.json(reviews);
+    } catch (error) {
+      res.status(500).json({ error: 'Failed to fetch reviews' });
```

Comment:
change the error message to add a period

Reply:
no can do!